kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/OperationSupport.java

```diff
@@ -493,6 +493,23 @@ protected Status handleDeploymentRollback(String resourceUrl, DeploymentRollback
     return handleResponse(requestBuilder, type, parameters);
   }
 
+  /**
+   * Sends a POST request to /eviction endpoint of specified resource
+   *
+   * @param evictionAsStr request body in form of JSON string(serialized form of Eviction)
+   * @param podUrl URL of Pod
+   * @throws IOException IOException
+   * @throws ExecutionException Execution Exception
+   * @throws InterruptedException Interrupted Exception
+   */
+  protected void handleEviction(String evictionAsStr, URL podUrl) throws IOException, ExecutionException, InterruptedException {
+    RequestBody requestBody = RequestBody.create(JSON, evictionAsStr);
+
+    URL requestUrl = new URL(URLUtils.join(podUrl.toString(), "eviction"));
+    Request.Builder requestBuilder = new Request.Builder().post(requestBody).url(requestUrl);
+    handleResponse(requestBuilder, null, Collections.emptyMap());
+  }
+
```

Comment:
- If this is specific to Pods, why did we add it to the base OperationSupport class?
- Can we accept a HasMetadata as parameter and do the conversion here?